tests/integration/workflows/nodejs_npm/testdata/npm-deps/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "npmdeps",
+  "version": "1.0.0",
+  "description": "",
+  "files": ["included.js"],
```

Comment:
Is this `files` section an advanced usecase? I felt like ppl usually just zipped up whatever is in their function's folder. 

If this is advanced, I want to make sure we cater to the basic users as well.

Reply:
perfect!